common/djangoapps/student/tests/test_enrollment.py

```diff
@@ -351,6 +351,14 @@ def test_with_invalid_course_id(self):
         resp = self._change_enrollment('unenroll', course_id="edx/")
         assert resp.status_code == 400
 
+    @patch.dict(settings.FEATURES, {'DISABLE_UNENROLLMENT': True})
+    def test_unenroll_disable_unenrollment(self):
+        """
+        Tests that a user cannot unenroll when unenrollment has been disabled
+        """
+        resp = self._change_enrollment('unenroll')
+        assert resp.status_code == 400
```

Comment:
@tinumide, can you add a check to verify that the user is still enrolled after this API call?